client/a8c-for-agencies/sections/partner-directory/agency-details/index.tsx

```diff
@@ -56,18 +54,18 @@ const AgencyDetailsForm = ( { initialFormData }: Props ) => {
 			reduxDispatch(
 				successNotice( translate( 'Your agency profile was submitted!' ), {
 					displayOnNextPage: true,
-					duration: 6000,
+					// duration: 6000,
```

Comment:
Is this intended? 🤔

Reply:
Good catch. I missed uncommenting this. Fixed it here: https://github.com/Automattic/wp-calypso/pull/97975/commits/e624754412c547e3f4d81e594f1cbc284f3fab9d